report.Rmd

```diff
@@ -10,6 +10,7 @@ params:
     language links that are similar to the individual search results. A
     control group will see the currently existing search results page.
   data: "./data/Explore similar pages, categories and suggested languages in search results/events_raw_20170724112938.RData"
+  query: "SELECT * FROM TestSearchSatisfaction2_16909631;"
```

Comment:
rmarkdown doesn't allow `query: NULL` in rmd file -- there has to be something instead of NULL. And we have to leave `query` there, otherwise we'll get error because there is `query` in `parameters.yaml` but not in report.rmd, even if `query: NULL` in `parameters.yaml`. Maybe `SELECT * FROM TestSearchSatisfaction2_16909631 LIMIT 10;`?